libs/auth/src/test_utils.ts

```diff
@@ -7,83 +7,42 @@ import type { Mocked, vi } from 'vitest';
 import { DippedSmartCardAuthApi } from './dipped_smart_card_auth_api';
 import { InsertedSmartCardAuthApi } from './inserted_smart_card_auth_api';
 
-type jfn = typeof jest.fn;
-type vfn = typeof vi.fn;
-
-/**
- * Builds a mock dipped smart card auth instance for application-level tests
- */
-export function buildMockDippedSmartCardAuth(): jest.Mocked<DippedSmartCardAuthApi>;
-/**
- * Builds a mock dipped smart card auth instance for application-level tests
- */
-
-export function buildMockDippedSmartCardAuth(
-  fn: jfn
-): jest.Mocked<DippedSmartCardAuthApi>;
-
-/**
- * Builds a mock dipped smart card auth instance for application-level tests
- */
-export function buildMockDippedSmartCardAuth(
-  fn: vfn
-): Mocked<DippedSmartCardAuthApi>;
-
 /**
  * Builds a mock dipped smart card auth instance for application-level tests
  */
 export function buildMockDippedSmartCardAuth(
-  fn: jfn | vfn = jest.fn
-): jest.Mocked<DippedSmartCardAuthApi> | Mocked<DippedSmartCardAuthApi> {
+  fn: typeof vi.fn
+): Mocked<DippedSmartCardAuthApi> {
```

Comment:
Do you think we'll eventually want to remove the `fn` argument from all of these mock builders and just import `vi.fn` directly from `vitest` in the defining module? Obviously less work to leave as is now that they are set up this way, but just wondering what your thoughts are

Reply:
Probably? If we did it that way we'd probably make `vitest` a `peerDependency`, right?

Reply:
Yeah that should work. I can't remember if we had it as a peer before, given that all the packages already have vitest as a dev dependency already.